src/simulator/views/gui/view_editor.py

```diff
@@ -137,6 +140,11 @@ def frame(self) -> DirectFrame:
     def marker(self) -> DirectFrame:
         return self.__marker
 
+    def enable(self) -> None:
+        self.__enabled = True
+
+    def disable(self) -> None:
+        self.__enabled = False
```

Comment:
Would it be better to have this as a `set_enabled(self, enabled: bool)` or similar?

Reply:
Yeah I suppose, changed.